dotnet/src/webdriver/DevTools/DevToolsSession.cs

```diff
@@ -600,7 +600,19 @@ private void ProcessMessage(string message)
                 var methodParts = method.Split(new char[] { '.' }, 2);
                 var eventData = messageObject["params"];
 
-                LogTrace("Recieved Event {0}: {1}", method, eventData.ToString());
+                if (eventData.AsObject().TryGetPropertyValue("request", out var requestNode)
+                    && requestNode.AsObject().ContainsKey("postData"))
+                {
+                    var loggableEventData = eventData.DeepClone();
+                    var loggableRequest = loggableEventData["request"]!;
+                    loggableRequest["postData"] = "*RAW POST DATA REMOVED FROM LOGS*";
+                    loggableRequest["postDataEntries"] = new JsonArray();
+                    LogTrace("Recieved Event {0}: {1}", method, loggableEventData.ToString());
+                }
+                else
+                {
+                    LogTrace("Recieved Event {0}: {1}", method, eventData.ToString());
+                }
```

Comment:
Fundamentally, this error happens because the string resembles JSON, but is in fact not JSON 
![image](https://github.com/user-attachments/assets/ed29be9e-4282-41a0-a659-271836ddff8f)